deploy.sh

```diff
@@ -3,6 +3,10 @@
 # Update all the submodules and translations
 ./update_submodules.sh
 
+yarn crawl
+git add "src/shared/instance_stats.ts"
+git commit -m "Crawl instance statistics"
```

Comment:
Wait a second, this seems wrong. Shouldn't it be that both the instance_stats.ts file should be left out of git, and the `yarn crawl` be run as part of the docker build? That way these stats aren't going to be a part of the git repo.

I ran `docker build . -t joinlemmy-site`, and because the crawl didn't run, it used old info.

Reply:
But then everyone who wants to develop this project needs to have a working Rust installation, and needs to have the crawler repository cloned in the right place. Otherwise they will just see an error, or maybe an empty instance list. I can change it so that this file is not committed, or just run the crawl explicitly in CI.